src/database/schema/ChannelListSchema.ts

```diff
@@ -185,10 +185,10 @@ class ChannelList implements BaseDbSchema {
           this.unreadCount,
           this.channelType,
           this.lastUpdatedAt,
-          this.lastUpdatedBy,
-          this.createdAt,
-          this.expiredAt,
-          this.topicPostExpiredAt,
+          this.lastUpdatedBy ?? null,
+          this.createdAt ?? new Date(),
+          this.expiredAt ?? null,
+          this.topicPostExpiredAt ?? null,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are to provide default values for `lastUpdatedBy`, `createdAt`, `expiredAt`, and `topicPostExpiredAt` properties. This is a good practice as it ensures that these properties will always have a value, preventing potential runtime errors due to undefined values.
```diff
-          this.lastUpdatedBy,
-          this.createdAt,
-          this.expiredAt,
-          this.topicPostExpiredAt,
+          this.lastUpdatedBy ?? null,
+          this.createdAt ?? new Date(),
+          this.expiredAt ?? null,
+          this.topicPostExpiredAt ?? null,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->